### PR TITLE
WEB-3255: Extend student list management on ccx_coach tab with the fu…

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** Extend student list management on ccx_coach tab with the full name column.

**JIRA:** https://liv-it.atlassian.net/browse/WEB-3255

screenshot from 2018-11-01 14-33-45

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] tag reviewer
- [ ] tag reviewer

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: themes/courses.labster.com/lms/templates/ccx/enrollment.html [hunk 1]
```diff
@@ -78,8 +78,15 @@ <h2> ${_("Student List Management")}</h2>
           </div>
         %endif
         <table>
+          <colgroup>
+            <col width="50">
+            <col width="70">
+            <col width="50">
+            <col width="35">
```
Comment (polesye) [inline themes/courses.labster.com/lms/templates/ccx/enrollment.html:85]: please set `<col width="1" />` to shrink by content

File: themes/courses.labster.com/lms/templates/ccx/enrollment.html [hunk 2]
```diff
@@ -78,8 +78,15 @@ <h2> ${_("Student List Management")}</h2>
...
+          </colgroup>
           <thead>
             <tr>
+              <td class="label">Name</td>
```
Comment (polesye) [inline themes/courses.labster.com/lms/templates/ccx/enrollment.html]: full name

File: themes/courses.labster.com/lms/templates/ccx/enrollment.html [hunk 3]
```diff
@@ -88,6 +95,7 @@ <h2> ${_("Student List Management")}</h2>
           <tbody>
             %for member in ccx_members:
             <tr>
+              <td>${member.user.profile.name}</td>
```
Comment (polesye) [inline themes/courses.labster.com/lms/templates/ccx/enrollment.html]: is it possible that profile related object doesn't exist?

File: themes/courses.labster.com/lms/templates/ccx/enrollment.html (hunk 1 above)
Comment (lukyfebrianto) [reply themes/courses.labster.com/lms/templates/ccx/enrollment.html:85]: It doesn't work, pls see the email column, 
screenshot from 2018-11-01 17-13-07

```
          <colgroup>
            <col width="1" />
            <col width="1" />
            <col width="1" />
            <col width="1" />
          </colgroup>
```

File: themes/courses.labster.com/lms/templates/ccx/enrollment.html (hunk 3 above)
Comment (lukyfebrianto) [reply themes/courses.labster.com/lms/templates/ccx/enrollment.html]: I think is possible if they create an account from edx admin or django console. In this case I add some checker for this kind situation.

File: themes/courses.labster.com/lms/templates/ccx/enrollment.html (hunk 2 above)
Comment (lukyfebrianto) [reply themes/courses.labster.com/lms/templates/ccx/enrollment.html]: changed

**Review by TamoshaytisV — APPROVED** [review]

**Review by polesye — APPROVED** [review]

**Pull request merged**